cmd/frcli/csv.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // CSVHeaders returns the headers used for harmony csv records.
-var CSVHeaders = "Timestamp,OnChain,Type,Category,Amount(Msat),Amount(USD),TxID,Reference,BTCPrice,BTCTimestamp,Note"
+var CSVHeaders = "Timestamp,OnChain,Type,Category,Amount(Msat),Amount(Fiat),TxID,Reference,BTCPrice,BTCTimestamp,FiatCurrency,Note"
```

Comment:
I think that we can be pretty certain that the report will always be generated with a single fiat currency, so we probably don't need to repeat it in a `FiatCurrency` column for each row. We could just insert our currency string in `Amount(Fiat)` where we [use this value](https://github.com/lightninglabs/faraday/blob/master/cmd/frcli/node_audit.go#L197).